pymc/gp/hsgp_approx.py

```diff
@@ -30,39 +31,42 @@
 TensorLike = np.ndarray | pt.TensorVariable
 
 
-def set_boundary(Xs: TensorLike, c: numbers.Real | TensorLike) -> TensorLike:
-    """Set the boundary using the mean-subtracted `Xs` and `c`.  `c` is usually a scalar
-    multiplyer greater than 1.0, but it may be one value per dimension or column of `Xs`.
+def set_boundary(Xs: TensorLike, c: numbers.Real | TensorLike) -> np.ndarray:
+    """Set the boundary using the mean-subtracted `Xs` and `c`. `c` is usually a scalar
+    multiplier greater than 1.0, but it may be one value per dimension or column of `Xs`.
     """
     S = pt.max(pt.abs(Xs), axis=0)
```

Comment:
Actually, shouldn't this also be using the midpoint @bwengals  (i.e `self._X_center`):
```python
S = (pt.max(Xs, axis=0) - pt.min(Xs, axis=0)) / 2
```
The current formula doesn't seem to actually compute the half range of X, does it?